orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/support/Capacity.groovy

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.pipeline.servergroup.support
+
+import groovy.transform.Canonical
+
+import javax.annotation.Nonnegative
+
+@Canonical
+class Capacity {
```

Comment:
As noted by `spinnakerbot` this should be Java...also probably can be an immutable class as well as I don't think we ever change the fields after creation.

Reply:
it's amazing how many `capacity` classes we have... If you can use an existing one (e.g. ResizeStrategy.Capacity)...
Or... we just go through and change all the other ones to use this one, since it's nicer anyway

Reply:
Yes the idea here was to create a more generic `Capacity` class instead of "stealing" the one from `ResizeStrategy`. Ideally it would also be a straight-up refactor but for some reason IntelliJ won't let me refactor an inner class in Groovy (?)

Reply:
How do we handle converting class to Java when the entire `orca-clouddriver` module is under `src/main/groovy`? Create a `src/main/java` with that lonely class in there?

Reply:
`.java` classes work fine even under `groovy` so I'd just put the `.java` class under `groovy` (looks like there are already a number of other java classes in there).